docs/en/tutorial.md

```diff
@@ -231,23 +246,21 @@ export default {
 
 There's a new object `vuex.getters` which requests `counterValue` to be bound to the getter `getCount`. We've chosen different names to demonstrate that you can use the names that make sense in the context of your component, not necessarily the getter name itself.
 
-You might be wondering, why is it preferable to use a getter instead of using something like `store.state.count` (which doesn't work, but still). While it would be ok here, in a large app:
+You might be wondering - why did we choose to use a getter instead of directly accessing the value from the state. This concept is more of a best practice, and is more applicable to a larger app, which presents several distinct advantages:
 
-1. A value may be derived from many other value (think totals, averages, etc.).
-2. Many components can use the same getter function.
+1. We may want to define getters with computed values (think totals, averages, etc.).
+2. Many components in a larger app can use the same getter function.
 3. If the value is moved from say `store.count` to `store.counter.value` you'd have to update one getter instead of dozens of components.
 
```

Comment:
we can also mention that getters are efficiently cached for performance (e.g. used by 100 components, but evaluated only once)